homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -142,23 +142,16 @@
 )
 
 
-async def async_setup_platform(
```

Comment:
this is a breaking change for existing `jewish_calendar` users because now the platform setup won't work. My suggestion is to keep this function around for some time (I believe the standard is one year), but the function forwards the config to config entry import

Reply:
@raman325 I understand now. Could you see if my latest changes resolve this issue?